### PR TITLE
fix(react/en) fix testing chapter link

### PR DESCRIPTION
**Pull request by geromegrignon**

### CODE REVIEW

File: content/react/en/addons.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ commit: "f4d933"
 # Addons
 
 Storybook boasts a robust system of [addons](https://storybook.js.org/addons/introduction/) with which you can enhance the developer experience for
-everybody in your team. If you've been following along with this tutorial linearly, we have referenced multiple addons so far, and you will have already implemented one in the [Testing chapter](/test).
+everybody in your team. If you've been following along with this tutorial linearly, we have referenced multiple addons so far, and you will have already implemented one in the [Testing chapter](../test).
```
Comment (tmeasday) [inline content/react/en/addons.md]: Generally we've used `(test)` rather than `(../test)`

**Comment by tmeasday** [comment]
Thanks for the pickup @geromegrignon !

File: content/react/en/addons.md (hunk 1 above)
Comment (geromegrignon) [reply content/react/en/addons.md]: Done!

**Comment by domyen** [comment]
Picking this back up after the holidays, @tmeasday is this OK to merge?

**Pull request closed without merging**